modules/apps/expando/expando-api/src/main/java/com/liferay/expando/util/ExpandoConverterUtil.java

```diff
@@ -197,8 +198,11 @@ else if (type == ExpandoColumnConstants.DATE_ARRAY) {
 				ArrayUtil.toStringArray((Date[])attribute, _getDateFormat()));
 		}
 		else if (type == ExpandoColumnConstants.STRING_LOCALIZED) {
-			return LocalizationUtil.getDefaultLocalizedString(
-				(Map<Locale, String>)attribute);
+			Locale defaultLocale = LocaleUtil.getDefault();
+
+			Map<Locale, String> values = (Map<Locale, String>)attribute;
+
+			return values.get(defaultLocale);
```

Comment:
Please squash this change with the previous.  It's ok if we lose some context as to how we eventually came to this solution, because this is clearly the "most correct" implementation, so the lost context isn't important.